solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -13,6 +14,7 @@ import "../../../token/interfaces/ISmartToken.sol";
 */
 contract LiquidityPoolV1Converter is LiquidityPoolConverter {
     IEtherToken internal etherToken = IEtherToken(0xc0829421C1d260BD3cB3E0F06cfE2D52db2cE315);
+    IOracle public oracle;

```

Comment:
Better move this to the converterBase. Will make it easier for us to not mess up existing structures with future updates.

Reply:
Ok